src/main/java/com/okta/tools/authentication/OktaMFA.java

```diff
@@ -120,13 +126,22 @@ private static JSONObject selectFactor(JSONObject primaryAuthResponse) throws JS
         }
 
         if (supportedFactors.size() > 1) {
+            if (environment.oktaMfaChoice != null) {
+                for (JSONObject factor : supportedFactors) {
+                    String provider = factor.getString("provider");
+                    String factorType = factor.getString("factorType");
+                    if ((provider + "." + factorType).equals(environment.oktaMfaChoice)) {
+                        return factor;
+                    }
+                }
+            }
```

Comment:
:seedling: core of the change. Halt MFA menu presentation and take the option based on config.properties/envvars if matches.